mailgun/views.py

```diff
@@ -10,3 +13,18 @@
 @require_http_methods(['GET'])
 def auth_error(request):
     return JsonResponse({'error': 'Failed to authenticate request.'}, status=401)
+
+
+@csrf_exempt
+@authenticate()
+@require_http_methods(['POST'])
+def log_post_data(request):
```

Comment:
If this was called `log_webhook()` it might be clearer what it's for (i.e. which post data it's collecting).